### PR TITLE
Fix bug in project management view

### PR DESCRIPTION
**Pull request by jrbotros**

### CODE REVIEW

File: orchestra/analytics/latency.py [hunk 1]
```diff
@@ -55,12 +55,17 @@ def work_time_df(projects, human_only=True, complete_tasks_only=True):
             get_project_information(project.id),
             human_only, complete_tasks_only)
         for project in projects))
-    df = DataFrame(time_data)
+
+    # Ensure that even an empty DataFrame has the correct columns
+    columns = ['assignment_level', 'calendar_time', 'end_datetime',
+               'iteration', 'project_description', 'project_id',
+               'start_datetime', 'task_id']
```
Comment (thisisdhaas) [inline orchestra/analytics/latency.py]: are these names spelled out elsewhere ever? if so, should they be a module level var `DATAFRAME_COLUMNS` somewhere?

File: orchestra/interface_api/project_management/project_management.py [hunk 2]
```diff
@@ -20,7 +20,6 @@ def project_management_information(project_id):
     project = Project.objects.get(id=project_id)
     df = work_time_df([project],
                       human_only=False, complete_tasks_only=False)
-    df['end_datetime'] = df['start_datetime'] + df['calendar_time']
     project_information = get_project_information(project.id)
```
Comment (thisisdhaas) [inline orchestra/interface_api/project_management/project_management.py:23]: why is this not necessary anymore?

**Comment by thisisdhaas** [comment]
LGTM

File: orchestra/analytics/latency.py (hunk 1 above)
Comment (marcua) [inline orchestra/analytics/latency.py]: if you do create a var, zip it with https://github.com/unlimitedlabs/orchestra/blob/master/orchestra/analytics/latency.py#L81 so we don't repeat the column names a bunch of times

File: orchestra/interface_api/project_management/project_management.py (hunk 2 above)
Comment (jrbotros) [inline orchestra/interface_api/project_management/project_management.py:23]: Was unnecessary to begin with :)

**Pull request merged**